### PR TITLE
Start building release "What's new" for release 1.8.X

### PR DESCRIPTION
**Pull request by gonidelis**
Added Pull Request and Documentation change description.

### CODE REVIEW

File: docs/sphinx/releases/whats_new_1_8_0.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+* :hpx-pr:`5776` - Revert 1 deepaksuresh1411 for loop patch
+* :hpx-pr:`5755` - Support for data-parallelism for mismatch algorithm
+* :hpx-pr:`5730` - fix returning non default constructable objects from action
+* :hpx-pr:`5708` - Use `/dev/shm` for Jenkins checkout on Piz Daint
+* :hpx-pr:`5705` - Update and add tests for `transfer/schedule_from`
+* :hpx-pr:`5703` - Fix `counting_iterator` container tests
+* :hpx-pr:`5689` - Enable LTO in CI
+* :hpx-pr:`5645` - Add matrix multiplication example
+* :hpx-pr:`5625` - Revert "Use advance_and_get_distance where required"
+* :hpx-pr:`5578` - Remove distributed functionality
```
Comment (hkaiser) [inline docs/sphinx/releases/whats_new_1_8_0.rst:50]: I believe @msimberg has a script that can generate this list directly from github.

Comment (gonidelis) [reply docs/sphinx/releases/whats_new_1_8_0.rst:50]: That's the (semi) output I get from that script. It's very bad, I asked him and wait for his response.

File: tools/roll_release.sh [hunk 2]
```diff
@@ -94,9 +94,8 @@ fi
 
 echo ""
 echo "Setting the signing key for signing the release. It is up to you to change it back to your own afterwards."
-git config user.signingkey E18AE35E86BB194F
-git config user.email "contact@stellar-group.org"
-git config user.name "STE||AR Group"
+git -c user.signingkey E18AE35E86BB194F -c user.email contact@stellar-group.org \
+    -c user.name "STE||AR Group"
```
Comment (msimberg) [inline tools/roll_release.sh:98]: @gonidelis the `-c` flags belong on the `git tag` call. From `man git`:
```
       -c <name>=<value>
           Pass a configuration parameter to the command. The value given will
           override values from configuration files. The <name> is expected in
           the same format as listed by git config (subkeys separated by
           dots).
```

Comment (gonidelis) [reply tools/roll_release.sh:98]: oh... my bad. that would be three `git tag` commands command one after the other, no?

Comment (gonidelis) [reply tools/roll_release.sh:98]: oh my bad...But I am still confused. What do you mean "belong"? I thought `git -c` is equivalent to `git config`, no?
So the proper code should be:

```
git -c user.signingkey E18AE35E86BB194F 
git -c user.email contact@stellar-group.org 
git -c user.name "STE||AR Group"
```
?

Comment (msimberg) [reply tools/roll_release.sh:98]: > @msimberg Or do I completely remove the `git config/-c` command and just embed `-c <name>=<value>` in the that line.

Yep!

**Comment by gonidelis** [comment]
@hkaiser  Should we patch the regression fails before merging?

**Comment by hkaiser** [comment]
@gonidelis is this ready to be merged now? If yes, let's do it.

**Comment by gonidelis** [comment]
> 

I would appreciate if:

1. you first revised that your what's new patch merged in properly (it's fine from what I see),
2. we could verify that @dimitraka's commits that are showing in my commit history do not create any conflicts/issues (this PR includes file changes that I haven't made, rather Dimitra did).
3. you verify that CI test.regressions fails are expected.

Then we can merge.

**Comment by hkaiser** [comment]
@gonidelis this patch goes towards master, so it will not influence the release, will it?

**Comment by hkaiser** [comment]
#5895 contains everything that's not already on master from this PR. Closing.

**Pull request closed without merging**